membership-attribute-service/app/utils/SimpleEitherT.scala

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import scalaz.{EitherT, \/}
```

Comment:
Can we use cats.EitherT instead please? I'd like to remove scalaz if possible

Reply:
That's a massive change since it's used everywhere.
I haven't introduced it, just aliased and renamed an existing type.

Reply:
If I was doing it though I'd switch everything, since it is all interlinked anyway.

Reply:
Some services in membership-common use EitherT.
And cats has no replacement for ListT.
So it is a bigger change.